ansible/roles_ocp_workloads/ocp4_workload_lpe_3scale/files/minio_s3_config.yml

```diff
@@ -0,0 +1,60 @@
+
+###
+# Minio
+# ---
+# Expect the Minio operator to already be setup
+# Provision the Minio instance
+
+- name: Include role 'ocp4-workload-minio'
+  include_role:
+    name: ocp4-workload-minio
+  vars:
+    _minio_project: "{{ ocp4_workload_lpe_3scale_minio_namespace }}"
+    _minio_storage_size: "{{ ocp4_workload_lpe_3scale_minio_pvc_size }}"
+    _minio_access_key: "{{ ocp4_workload_lpe_3scale_minio_access_key }}"
+    _minio_secret_key: "{{ ocp4_workload_lpe_3scale_minio_secret_key }}"
+
+- name: Get external Minio instance route
+  k8s_info:
+    api_version: route.openshift.io/v1
+    kind: Route
+    name: "{{ ocp4_workload_lpe_3scale_minio_route_name }}"
+    namespace: "{{ ocp4_workload_lpe_3scale_minio_namespace }}"
+  register: _minio_route
+  retries: 3
+  delay: 5
+  until: _minio_route.resources[0].spec.host != ''
+
+- set_fact:
+    _minio_hostname: http://{{ _minio_route.resources[0].spec.host }}
+
+
+
+- shell: AWS_ACCESS_KEY_ID={{ ocp4_workload_lpe_3scale_minio_access_key }} AWS_SECRET_ACCESS_KEY={{ ocp4_workload_lpe_3scale_minio_secret_key }} aws s3api --no-verify-ssl --endpoint-url {{ _minio_hostname }} --region {{ ocp4_workload_lpe_3scale_minio_bucket_region }} create-bucket --bucket {{ ocp4_workload_lpe_3scale_minio_bucket_name }}
```

Comment:
You can use `environment:` on a task to set the environment variables. No need to pass them like this. That will also keep the keys out of the logs. Also the task should have a name. Finally this is going to fail YAML Lint - use `>-` to be able to split the line into multiple lines.

I guess there is no module for s3 is there?